src/lib.rs

```diff
@@ -12,11 +15,12 @@ impl Default for RetentionMode {
 }
 #[derive(Clone, Debug)]
 pub struct SleepParams {
-    pub sleep_period: u64,
-    pub max_sleep_period: u64,
-    pub min_sleep_period: u64,
-    pub sleep_step: u64,
+    pub sleep_period: Duration,
```

Comment:
looks great. now readme should be updated

Reply:
[Readme updated!](https://github.com/ayrat555/fang/pull/67/commits/8a0c4b5cea557d28f204cb51028ad8aa9bab0dc1)